app/src/test/java/org/mozilla/fenix/tabstray/DefaultTabsTrayControllerTest.kt

```diff
@@ -543,6 +544,91 @@ class DefaultTabsTrayControllerTest {
         }
     }
 
+    @Test
+    fun `GIVEN no tabs selected and the user is in multi select mode WHEN the user long taps a tab THEN that tab will become selected`() {
```

Comment:
```suggestion
    fun `GIVEN no tabs selected and the user is not in multi select mode WHEN the user long taps a tab THEN that tab will become selected`() {
```

doesn't this test cover the first scenario of the `when` in `handleMultiSelectClicked` when the user is *not* in multi select mode?